xyzspaces/spaces.py

```diff
@@ -199,11 +212,6 @@ def update(
         ...              description="updated description",
         ...              tagging_rules=tagging_rules)
         """
-        if not (title or description or tagging_rules or schema):
-            raise ValueError(
-                "Please provide either title, description or tagging rules to"
-                " or schema to update."
-            )
```

Comment:
As you have removed the if condition then if None of the value is passed then it might cause an issue

Reply:
passing all default values does not cause any issue. as space is not updated. I have added test for this, this is covered in test `test_unshare_space`

Reply:
Ok Cool